src/task6/MarkArray.java

```diff
@@ -0,0 +1,56 @@
+package task6;
+/*Создать массив оценок произвольной длины, вывести максимальную и минимальную оценку, её (их) номера.*/
```

Comment:
I've just done. Please find the second way to resolve the task in my 2nd commit.
But as far as I understand the recommended aproach with method Array.sort makes code more complicated. 
We need to make a new object - new array and make an additional variable. Because we should not only to find max and min values but also we have to find their index or indexes (if the number of marks with max/min value is more than 1). 
Using method Array.sort elements of initial array changes their "place" in array and their indexes as well. After the implementation this method we can't get the right index(es). So we should create a duplicate of initial array.
If I was wrong I'd apreciate it if you could share with me the algorithm mentioned by you that can simplify code.
Many thanks in advance.

Reply:
But I don't see a condition specifying the requirement to specify array indexes.